<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1330</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MediaPlay</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="17">
     <widget class="QLabel" name="label_Total_Time">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>00:00:00</string>
      </property>
     </widget>
    </item>
    <item row="2" column="15">
     <widget class="QPushButton" name="pushButton_Volume">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QLabel" name="label_current_Time">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>00:00:00</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_Seek_Backward">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_Seek_Forward">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="16">
     <widget class="QSlider" name="horizontalSlider_Duration">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>910</width>
        <height>0</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::NoFocus</enum>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="pageStep">
       <number>0</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TickPosition::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="18" rowspan="3">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
      </property>
      <property name="indentation">
       <number>15</number>
      </property>
      <property name="allColumnsShowFocus">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="headerHighlightSections">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">Content</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0" colspan="18">
     <widget class="QGroupBox" name="groupBox_Video">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>1080</width>
        <height>720</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox
{
        border:1px solid black;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="16">
     <widget class="QSlider" name="horizontalSlider_Volume">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_Play_Pause">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_Stop">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1330</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSelect_Video_File">
    <property name="title">
     <string>Select Videos Folder</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="playInViewLOg_check_box"/>
   </widget>
   <addaction name="menuSelect_Video_File"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="playInViewLOg_check_box">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Play Media in ViewLog</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
